.moai/project/structure.md

```
@@ -873,29 +712,31 @@ Converged? --> Yes: Complete
 | Language Servers | JSON-RPC 2.0 (stdio/TCP) | `internal/lsp/` | Code diagnostics |
 | Git | system Git via `exec.Command` | `internal/core/git/` | Version control |
 | ast-grep | CLI subprocess | `internal/astgrep/` | Structural code analysis |
-| Ranking API | HTTPS REST | `internal/rank/` | Performance metrics |
+| GitHub API | `gh` CLI subprocess | `internal/github/` | Issue/PR automation, spec linking |
 | GitHub Releases | HTTPS REST | `internal/update/` | Self-update |
-| File system | OS syscalls | `pkg/utils/` | Configuration, templates |
+| File system | OS syscalls | `internal/atomicfile/`, `internal/manifest/` | Atomic writes, provenance tracking |
 
 ### Internal Module Dependencies
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)